containers/Red/README.md

```diff
@@ -0,0 +1,53 @@
+# Red Docker image
+
+Dockerfile for generating a Docker image for Red, "an intelligent, rapid, accurate tool for detecting repeats de-novo on the genomic scale".
+
+Links:
+[source code](https://github.com/BioinformaticsToolsmith/Red)
+[publication](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0654-5)
+
+
+Docker image at Docker Hub:
+https://hub.docker.com/r/williamebi/red
+
+
+## build, publish, and run a Docker image
+
+build a Docker image
+```
+docker image build -t williamebi/red:2.0 .
+```
+
+run the Docker image locally
+```
+# print the help message
+docker run williamebi/red:2.0
+
```

Comment:
I think this is something we need to make sure that every one creating an image need to have the help printed when running without argument

Reply:
Yes, it would be really helpful.